web-ui/src/context/selectors.js

```diff
@@ -40,6 +40,12 @@ export const selectIsAdmin = createSelector(
     userProfile && userProfile.role && userProfile.role.includes("ADMIN")
 );
 
+export const selectHasReportPermission = createSelector(
+    selectUserProfile,
+    (userProfile) =>
+        userProfile && userProfile.role && userProfile.permissions.any((permission) => permission.includes("REPORT"))
+);
+
```

Comment:
@ZacharyKlein permissions for the current user are already loaded in state?

Reply:
@mkimberlin I believe so, when I inspect the network response from `/member-profiles/current` contains this block:

```
"permissions": [
        {
            "id": "008f6641-0b0a-4e89-84f0-c580f912b80d",
            "permission": "CAN_VIEW_FEEDBACK_REQUEST"
        },
        {
            "id": "056e32b2-8d07-11ee-b9d1-0242ac120002",
            "permission": "CAN_VIEW_ANNIVERSARY_REPORT"
        },
```